test/system/test_tpm2_quote.sh

```diff
@@ -77,7 +77,14 @@ tpm2_takeownership -c
 
 tpm2_createprimary -Q -A e -g $alg_primary_obj -G $alg_primary_key -C $file_primary_key_ctx
 
-tpm2_create -Q -g $alg_create_obj -G $alg_create_key -u $file_quote_key_pub -r $file_quote_key_priv  -c $file_primary_key_ctx
+# Certain TPM 2.0 chip, e.g, Nationz Z32H320TC, and tpm2simulator referring
+# to spec 1.16 may have an errata, disallowing both decrypt and sign set for
+# a symcipher object, and in response to RC_ATTRIBUTES. In order to work
+# around it, we attempt to run tpm2_create again with sign bit clear.
+tpm2_create -g $alg_create_obj -G $alg_create_key -u $file_quote_key_pub -r $file_quote_key_priv -c $file_primary_key_ctx 2>&1 |
+    grep -q 'Create Object Failed ! ErrorCode: 0x2c2' && {
+        tpm2_create -g $alg_create_obj -G $alg_create_key -A 0x20072 -u $file_quote_key_pub -r $file_quote_key_priv -c $file_primary_key_ctx
+    }
```

Comment:
if your copying and pasting this issue arround, it looks like maybe a test_helpers.sh that can be sourced is prudent, and then you just call that common piece. This way, when error codes get cleaned up, were not changing it in N > 1 spots.